### ISSUE TITLE
erm exe file where?

### ISSUE THREAD

**Issue by SoullessRand0** [keenthinker/WindowsLockScreenImages#1]
README says to run the executable file, but couldnt find 1

**Comment by keenthinker** [comment]
Hi, thanks for the remark. I'm currently working on the actions to automatically create the execution files. I'll inform you once the executables are available.

**Comment by DCullenV** [comment]
Same issue here... I don't see the executable mentioned in the instructions.

**Comment by keenthinker** [comment]
Hi @SoullessRand0 ,
Hi @DCullenV ,

I just created the first release v1.0.0 and uploaded the binaries. You will find the files here: [https://github.com/keenthinker/WindowsLockScreenImages/releases](https://github.com/keenthinker/WindowsLockScreenImages/releases)

I added a GitHub action to the project to build the executable and create a ZIP file with all needed files. The executable is build on the GitHub servers (and not on my machine) and uploaded as a release.

I'll be happy to answer any questions and or hear your feedback. Thanks. 

Best regards
